test/widgets/action_sheet_test.dart

```diff
@@ -472,6 +476,14 @@ void main() {
       await tapCopyMessageTextButton(tester);
       await tester.pump(Duration.zero);
       check(await Clipboard.getData('text/plain')).isNotNull().text.equals('Hello world');
+
+      // regression check for #732
+      final snackbar = tester.widget<SnackBar>(find.byType(SnackBar));
```

Comment:
This test (and the next test for CopyMessageLinkButtion) still passes with the old code where the wrong `BuildContext` is passed! In the test environment, that context seems to be mounted even `Navigator.of(context).pop()` is called before.